packages/generator-react-server/test/app.js

```diff
@@ -24,6 +24,46 @@ test('generator-react-server:app creates default files', async t => {
 	t.false(await exists('docker-compose.yml', testDir));
 });
 
+test('generator-react-server:app .reactserverrc matches react-server-cli default option keys', async t => {
+	let testDir;
+	await helpers.run(path.join(__dirname, '../generators/app'))
+		.inTmpDir(dir => {
+			testDir = dir;
+		})
+		.withPrompts({name: 'foo', dockerCfg: false})
+		.toPromise();
+
+	// Reading files here instead of requiring because we don't understand export default here.
+	const defaultOptions = await readFile('defaultOptions.js', path.join(__dirname, '../../react-server-cli/src'));
```

Comment:
Oh, I see.  Yeah, that's the source file, which hasn't gone through babel yet.

I think the nicest here would be to export `defaultOptions` from `react-server-cli`?

Then you could just do:

```javascript
import {defaultOptions} from "react-server-cli";
```

If you're up for that, the CLI exports are set up [here](https://github.com/redfin/react-server/blob/master/packages/react-server-cli/src/index.js#L20-L23).  Because the CLI has a build step that does babel transpilation you'll need to cd in there and `npm run prepublish` for the changes to become available.